lib/fs.js

```diff
@@ -221,19 +191,17 @@ fs.Stats.prototype.isSocket = function() {
 });
 
 fs.access = function(path, mode, callback) {
+  callback = makeCallback(arguments[arguments.length - 1]);
```

Comment:
hmm... not thrilled about this approach. For instance, the following is technically not an incorrect way of calling the `fs.access()` function even if the extraneous arguments on the end aren't supported. This code, however, would fail:

```js
fs.access('/some/path', () => {}, 'useless', 'arguments', here');
```